lib/Installation/Popups/InstallBackPopupController.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
```

Comment:
We don't need this controller, we just need to use OKCancelPopup inside InstallationSettingsController (which currently is called OverviewController) and have there methods `install()` and `confirm_install()` It is part of that screen even if it is a popup, so we can still keep a popup generic that we can use everywhere and have nice interface method to say from the test, install or decline install.